sdk/schemaregistry/azure-data-schemaregistry-apacheavro/src/main/java/module-info.java

```diff
@@ -2,8 +2,8 @@
 // Licensed under the MIT License.
 
 module com.azure.data.schemaregistry.apacheavro {
+    requires transitive com.azure.core;
```

Comment:
Is this required? I think transitively requiring `com.azure.data.schemaregistry` should include `com.azure.core` too.

Reply:
Yeah. I had an error when I didn't. :/